tidy3d/components/simulation.py

```diff
@@ -3217,3 +3218,49 @@ def perturbed_mediums_copy(
             sim_dict["medium"] = med.perturbed_copy(**restricted_arrays)
 
         return Simulation.parse_obj(sim_dict)
+
+    @cached_property
+    def scene(self) -> Scene:
```

Comment:
is this implemented in the super class?

Reply:
same for `from_scene`? also add from_scene return type

Reply:
actually, currently `Simulation` doesn't inherit from the new `AbstractSimulation` per our discussion to leave it on its own for a bit (similarly, `SimulationData`, monitors, etc). But I would like to do this transition in the next PR.